src/test/unit/services/pathfinder/psis_test.cpp

```diff
@@ -0,0 +1,179 @@
+#include <stan/services/pathfinder/psis.hpp>
+#include <test/unit/services/pathfinder/util.hpp>
+#include <gtest/gtest.h>
+
+auto&& blah = stan::math::init_threadpool_tbb(1);
+
+TEST(ServicesPSIS, xl) {
+  Eigen::Array<double, -1, 1> test_x(20);
+  test_x << 0.00231135747917145, 0.00433831801177895, 0.0108541508266367,
+      0.0146361066006147, 0.016979809437058, 0.0175260143161184,
+      0.0185788657942587, 0.0221000771645818, 0.0246924617215394,
+      0.0274311810928377, 0.0492474466172383, 0.0623614301581945,
+      0.0659810585631264, 0.0796802961280512, 0.146305816395337,
+      0.184840740340265, 0.20479110080993, 0.257155687804798, 0.414031661251632,
+      0.95242504763768;
+  Eigen::Array<double, -1, 1> theta(34);
+  theta << -141.2015714797, -72.7818997030625, -51.7150259992074,
+      -40.5047923559694, -33.2797837849548, -28.1283549132001,
+      -24.2170675460217, -21.1168185810503, -18.5812072879574,
+      -16.4573241245016, -14.6445871694482, -13.0737624253249,
+      -11.6954262460058, -10.4732265260756, -9.37974923155375,
+      -8.39387784885848, -7.49904768025677, -6.68205735201866,
+      -5.93223884538201, -5.24086486593837, -4.60071729862994,
+      -4.00576743436348, -3.45093529145725, -2.93190590437957,
+      -2.44498730074187, -1.98699942882981, -1.55518636773351,
+      -1.14714626375751, -0.760774912681297, -0.394219954206013,
+      -0.045843397405553, 0.285809256288784, 0.602034622751884,
+      0.903992807508582;
+  auto xx
+      = stan::services::psis::internal::profile_loglikelihood(theta, test_x);
+  // std::cout << "\nxx: \n" << xx << "\n";
+  Eigen::Array<double, -1, 1> good_vals(34);
+  good_vals << 1.06041260401414, 1.15443356538553, 1.19147065735959,
```

Comment:
can you mention where the good_vals came from as a comment?

Reply:
Fixed